frontend/src/component/pages/HomePage.tsx

```diff
@@ -108,11 +112,19 @@ export class HomePage extends Component<Props, State> {
     this.setState({
       isUserSignedIn: true
     });
+    this.setUserAdminStatus();
     this.handleStateChange();
     this.autoFillLongLink();
     this.autoShowChangeLog();
   }
 
+  private setUserAdminStatus = async () => {
+    const isUserAdmin = await this.props.shortHTTPApi.isUserAdmin();
+    this.setState({
+      isUserAdmin: isUserAdmin
+    });
```

Comment:
I wonder whether we can call “includeAdminPage” instead of adding a new function on HTTPApi? It seems that `isUserAdmin` is a feature decision and not a seperate HTTP call